osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs

```diff
@@ -141,6 +144,10 @@ protected override void LoadComplete()
 
             arrow.Alpha = 0;
             arrow.Delay(200).FadeInFromZero(600);
+
+            var progressBar = Parent.ChildrenOfType<SongProgress>().FirstOrDefault();
```

Comment:
`ChildrenOfType` is *never* used outside of testing methods. Also we don't do cross-component dependencies like this. `SongProgress` is not responsible for seeking, so shouldn't be the event source for other components to read from.

You should be resolving `GameplayClock` or `Player` or something and working from there.

Reply:
Resolved in latest commit